test/unit/alignment/configuration/align_config_edit_test.cpp

```diff
@@ -9,11 +9,11 @@
 
 #include <seqan3/alignment/configuration/align_config_edit.hpp>
 
-TEST(align_cfg_edit, is_global)
-{
-    using type_of_cfg = decltype(seqan3::get<seqan3::align_cfg::mode>(seqan3::align_cfg::edit).value);
-    EXPECT_TRUE((std::is_same_v<std::remove_reference_t<type_of_cfg>, seqan3::detail::global_alignment_type>));
-}
+// TEST(align_cfg_edit, is_global)
+// {
+//     using type_of_cfg = decltype(seqan3::get<seqan3::align_cfg::method>(seqan3::align_cfg::edit).value);
+//     EXPECT_TRUE((std::is_same_v<std::remove_reference_t<type_of_cfg>, seqan3::detail::method_global_tag>));
+// }
```

Comment:
Explain in a comment why this is commented out and when it can be commented in again (maybe once an issue is resolved?)

Reply:
This was actually a missed WOP, thank you for pointing it out. The test is now adopted for the method configuration.